README.md

```diff
@@ -231,6 +231,11 @@ You could pass it like:
             selectPage=(action "selectCurrentPage")
             clearSelection=(action "clearSelect")}}
 ```
+If you want to use the build-in selection mechanism, but without using an actionList, you can pass the following property:
+- `selectionEnabled`
```

Comment:
Yeah, I like this name. Unfortunately you didn't use this, but made it `enableSelection`